shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/security/AesAlgorithmSecure.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.infra.security;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+import java.security.SecureRandom;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.List;
+
+/**
+ * this is Aes Security Utils for encrypt or decrypt password
+ * NOTICE:
+ * in encrypt, we random get iv and key in 16-bytes to encrypt password, but in return value, we combain
+ * iv + key + encrypt(passwd) as a string, this is not SAFE!!! iv and key must save other security file or database!
+ * if you need more security, please implements AlgorithmSecure youself.
+ * in decrypt, we first extract iv and key in 16 bytes, then use it to decrypt password.
+ */
+public class AesAlgorithmSecure implements AlgorithmSecure {
+    // the iv and key lenght
```

Comment:
Can you remove the comment?
Only JavaDoc and TODO permitted

Reply:
fixed.